app/views/playlist_items/_markers.html.erb

```diff
@@ -58,7 +61,7 @@ Unless required by applicable law or agreed to in writing, software distributed
                 <% end %>
               <% end %>
               <%# Delete button %>
-              <% if can? :delete, marker %>
+              <% if showActions %>
```

Comment:
Now that we're relying on this boolean flag instead of the two `can?` checks, could the two `if` blocks be collapsed into one?

Reply:
sure, i was just waiting for someone to say that or anything else :-)